adresse/controllers/service.classic.php

```diff
@@ -165,4 +169,92 @@ function update(){
     $rep->data = array('success' => ''.$result, 'type'=>$typeRes, 'message' => $message);
     return $rep;
   }
+
+  function export(){
+
+    $test='#^[0-9]{1}[0-9AB]{1}[0-9]{3}$#';
+    $filterParams = array();
+    $project = $this->param('project');
+    $repository = $this->param('repository');
+    $insee = $this->param('insee');
+    $option = $this->param('opt');
+
+    $rep = $this->getResponse('json');
+    if(!$project){
+      $rep->data = array('status'=>'error', 'message'=>'Project not find');
+      return $rep;
+    }
+
+    if(!$repository){
+      $rep->data = array('status'=>'error', 'message'=>'Repository not find');
+      return $rep;
+    }
+
+    if(!$insee){
+      $rep->data = array('status'=>'error', 'message'=>'Code insee not find');
+      return $rep;
+    }
+    if(!preg_match($test, $insee)){
+      $rep->data = array('status'=>'error', 'message'=>'Code insee not avaible');
```

Comment:
```suggestion
      $rep->data = array('status'=>'error', 'message'=>'Code insee not available');
```

Je corrige en anglais la faute. Mais si le projet est franco-français et que l'on ne traduit pas par la suite l'application, je pense que nous devons passer en français les messages.

Et pour les lignes au dessus/dessous, les messages d'erreurs en anglais, c'est plutôt `not found`, pas `not find` ;-)